### DISCUSSION TITLE
Update Installation instructions on Readme

### DISCUSSION THREAD

**Discussion by dcaswel** [worksome/request-factories#11] [Ideas]
So I followed the directions in the Readme to install the package as a dev dependency. This will work fine as long as you are not planning on using the `HasFactory` trait. If you do use the trait, when you run install for Prod meaning without dev dependencies then PHP is going to throw an error when it tries to read the trait that isn't there. The Readme should probably be updated to point out that if you are going to use that trait then you need to install the package as a regular dependency instead of a dev dependency. If I get a minute I will create a PR for this but I just wanted to point it out in case someone is able to get to it before I can.

**Comment by lukeraymonddowning** [comment]
Thanks! Updated the README 👍
